src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+if (thumbs && largeImg) {
+  thumbs.addEventListener('click', handleThumbnailClick);
+}
+
+function handleThumbnailClick(e) {
+  e.preventDefault();
+
+  const link = e.target.closest('.list-item__link');
```

Comment:
The `e.target.closest('.list-item__link')` might return null if the click event is not on an element with the class 'list-item__link' or its descendant. Ensure that the HTML structure supports this logic.